extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-service.ts

```diff
@@ -127,11 +128,11 @@ export class ChePluginServiceImpl implements ChePluginService {
                 const httpOverHttpsAgent = tunnel.httpOverHttps({ proxy: httpsProxyOptions });
```

Comment:
It seems to me that the `httpsProxyOptions` might need to have the `public-certs` as well, since this is the bundle that contains the intercepting proxy CA (coming from the cluster-wide-proxy).

Does it make sense @tolusha ?

The same change would have to be done on the corresponding changes in the `che-workspace-client` library.